utils/huggingface/converters.py

```diff
@@ -46,6 +47,10 @@ def write_asset(api_key, job_name, labels_to_ids, handler, asset):
         },
     )
     text = response.text
+    if job_name not in asset["labels"][0]["jsonResponse"]:
```

Comment:
We need to create a dataclass Asset in a next ticket. Why do we take the first element with `[0]`? Are we sure the jsonResponse does not vary if we take the second element ?

Reply:
this is completely arbitrary, I'll add a comment. In the future, we may add a "merge" strategy for the assets that have several labels (here, this would be the `first` strategy). Adding a ticket for that.

Reply:
For the assets class, yes why not